include/miopen/config.h.in

```diff
@@ -152,4 +152,6 @@
 #include <hip/hip_version.h>
 #endif
 
+#define MIOPEN_GOLDEN_DB_VERSION 5
```

Comment:
@cderb Hi Chris, AFAICS this macro is not used in the library. Can you please shed some light on what it is for?

Reply:
@cderb have the same question

Reply:
Tuna is now managing a "golden table" for MIOpen tunings which is a set of versioned databases for MIOpen. This define refers to the version of that table currently in MIOpen.